pkg/gui/gui.go

```diff
@@ -346,6 +346,9 @@ type guiState struct {
 
 	// for displaying suggestions while typing in a file name
 	FilesTrie *patricia.Trie
+
+	// this is the message of the last failed commit attempt
+	messageFailedCommit string
```

Comment:
could we rename this to `failedCommitMessage`? Just so that the reader doesn't believe that an actual commit is stored in this variable

Reply:
Done